akka-bbb-apps/src/main/scala/org/bigbluebutton/core2/message/handlers/GetMeetingInfoMsgHdlr.scala

```diff
@@ -0,0 +1,76 @@
+package org.bigbluebutton.core2.message.handlers
+
+import org.bigbluebutton.core.models.{ RegisteredUsers, Users2x, VoiceUsers }
+import org.bigbluebutton.core.models.VoiceUsers.findAllListenOnlyVoiceUsers
+import org.bigbluebutton.core.models.Webcams.findAll
+import org.bigbluebutton.core.running.MeetingActor
+import org.bigbluebutton.core2.MeetingStatus2x
+import org.bigbluebutton.core2.MeetingStatus2x.hasAuthedUserJoined
+import org.bigbluebutton.protos.{ BreakoutInfo, DurationInfo, MeetingInfo, ParticipantInfo, User }
+
+trait GetMeetingInfoMsgHdlr {
+  this: MeetingActor =>
+
+  def handleGetMeetingInfo(): MeetingInfo = {
+    val users = for {
+      u <- Users2x.findAll(liveMeeting.users2x)
+    } yield {
+      User(
+        userId = u.intId,
+        fullName = u.name,
+        role = u.role,
+        isPresenter = u.presenter,
+        isListeningOnly = findAllListenOnlyVoiceUsers(liveMeeting.voiceUsers).exists(v => v.intId == u.intId),
+        hasJoinedVoice = VoiceUsers.findAllNonListenOnlyVoiceUsers(liveMeeting.voiceUsers).exists(v => v.intId == u.intId),
+        hasVideo = findAll(liveMeeting.webcams).exists(w => w.userId == u.intId),
```

Comment:
afaik we support more than one webcam per user. Would it be of any utility to expose this info as a number of webcams rather than boolean?